librocksdb-sys/Cargo.toml

```diff
@@ -9,16 +9,18 @@ readme = "README.md"
 repository = "https://github.com/rust-rocksdb/rust-rocksdb"
 keywords = [ "bindings", "ffi", "rocksdb" ]
 
+build = "build/main.rs"
```

Comment:
Does it really make sense ? Why just don't use `build.rs`?

Reply:
So I can split vendoring and system linking in separate files. Vendoring is really complex, linking is really simple, having them live together helps make the build system seem obscure. The way the splitting was done makes, in my opinion, the life of a new contributor much easier.

Reply:
I could also have left `build.rs`, `vendor.rs`, and `system.rs` in the top-level, but I think that’s somehow even weirder. Alternatively I could just rename `build/main.rs` to `build/build.rs`, if you think that’s better :)

Reply:
Move logic in different files it's ok. I mean why to rename `build.rs` to `main.rs` and define in `Cargo.toml` that content of build script now is placed in `main.rs` 🙃

Reply:
Fixed!